Vagrantfile

```diff
@@ -3,6 +3,8 @@ require 'ipaddr'
 
 @ui = Vagrant::UI::Colored.new
 
+K8S_PLAYGROUND_VERSION="2.0.1"
```

Comment:
We should get this dynamically from the git tag, and if the current commit is newer then a tag, we should print the commit hash instead. Let me script the logic for you.

Reply:
you are very welcome to script this feature ! I do not know how to do it...

we should also add this check to travis:

```
if over-engineering-factor <= 2 :
  fail
```

:-D

Reply:
I already see us (=me) opening issues because we forgot to update that number :D

Reply:
removed !